File: python/series.py
Count = int(input('Enter the number to print series of a number with the power of same number upto given number:'))
print "The series of number with power of same number upto " + str(Count) + ": ",
Number = 1
Product = 1
while Product <= Count:
  Counter = 1
  Product = 1
  while Counter <= Number:
    Product = Product * Number
    Counter = Counter + 1
  if Product <= Count:
    print str(Product),
  Number = Number + 1
  
print('.')
    

File: java/series.java
import java.util.*;
class Main {
  public static void main(String[] args) {
    int Count, Counter, Product, Number;
    Scanner scan = new Scanner(System.in);
    System.out.print("Enter the number to print series of number with the power same number upto given number: ");
    Count = scan.nextInt();
    System.out.print("The series of number with power of same number upto " + Count + ": ");
    Number = 1;
    Counter = 1;
    Product = 1;
    while(Product < Count){
      while(Counter <= Number){
        Product = Product * Number;
        Counter++;
      }
      if(Product < Count){
        System.out.print(" " + Product);
      }
      Number++;
    } 
    System.out.print(".");
  }
  
}